spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -350,4 +350,20 @@ public void deleteItem(String groupId, String citationKey, Long citationVersion)
         }
 
     }
+    
+    @Override
+    public void deleteMultipleItems(String groupId, String citationKeys, Long citationVersion) throws ZoteroConnectionException {
+        String url = String.format("groups/%s/%s", groupId, "items?itemKey=" + citationKeys);
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("If-Unmodified-Since-Version", citationVersion + "");
+
+        HttpEntity<JsonNode> dataHeader = new HttpEntity<JsonNode>(headers);
+
+        try {
+            restTemplate.exchange(buildUri(url, false), HttpMethod.DELETE, dataHeader, new ParameterizedTypeReference<String>() {});
```

Comment:
does this api call not get a response that indicates if it was successful? How does the caller know everything went well?